<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAircraftModelForm</class>
 <widget class="QFrame" name="CAircraftModelForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aircraft model form</string>
  </property>
  <layout class="QHBoxLayout" name="vl_AircraftModelForm" stretch="1,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="fr_AircraftModelLeft">
     <layout class="QVBoxLayout" name="vl_AircraftModelFormLeft">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="swift::gui::editors::CDistributorForm" name="editor_Distributor">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="swift::gui::editors::CAircraftIcaoForm" name="editor_AircraftIcao">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr_LeftEmpty">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fr_AircraftModelRight">
     <layout class="QVBoxLayout" name="vl_AircraftModelFormRight">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="swift::gui::editors::CLiveryForm" name="editor_Livery">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr_RightEmpty">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::editors::CAircraftIcaoForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/aircrafticaoform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::editors::CLiveryForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/liveryform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::editors::CDistributorForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/distributorform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
